src/app/code/community/Omise/Gateway/controllers/Checkout/PaynowController.php

```diff
@@ -0,0 +1,9 @@
+<?php
+class Omise_Gateway_Checkout_PaynowController extends Mage_Core_Controller_Front_Action
+{
+    public function indexAction()
```

Comment:
This is the same as Tesco - should probably share a common parent for these 'barcode' related payments